### PR TITLE
support for touch devices

### PR DESCRIPTION
**Pull request by daeks**
ref. to https://github.com/Codiad/Codiad/issues/772 and https://github.com/Codiad/Codiad/pull/780

### CODE REVIEW

**Comment by daeks** [comment]
Please test on several devices and add results to this PR.

Currently verified:

- Chrome for iOS on an iPad

**Comment by daeks** [comment]
someone else could ttest that? Due I have no device anymore, I am unable to process with that. Otherwhise I would like to close it.

**Comment by basteln3rk** [comment]
I can confirm that opening files by tapping now works, tested on Chrome 36/Android 4.4!

What else works
----------------------
* opening and closing folders in the files navigation
* switching between open windows
* opening the right side bar

What does not work
--------------------------
* i cannot hide the location bar by scrolling down. that's a shame, because the screen space would have been really useful.
* clicking the "x" to close an open file in a tab
* closing the right sidebar: when I click on this icon in openend state, nothing happens
* and, perhaps, most fundamentally: editing text. the keyboard gets very confused, probably by the JS handing cursors - for example, a space becomes a backspace on my device (I use SwiftKey, maybe there is a relationship)

So in conclusion: the PR succeeds in its purpose (enabling files to be opened on mobile devices), and I'd be happy to see it merged. This makes Codiad suitable as a mobile file viewer at least.

In the future, there are some more enhancements that could make Codiad a really great option for "emergency" code access from a mobile
